From 3ab6873c44ad7062bcc5d83d88dbb52e2678d991 Mon Sep 17 00:00:00 2001
From: Philippe Vaucher <philippe.vaucher@gmail.com>
Date: Fri, 8 May 2020 17:35:44 +0200
Subject: [PATCH] Fix warning preventing build on ubuntu 14.04

---
 lib/stdio.in.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/stdio.in.h b/lib/stdio.in.h
index ce00af574a..3964811f62 100644
--- a/lib/stdio.in.h
+++ b/lib/stdio.in.h
@@ -715,7 +715,6 @@ _GL_CXXALIASWARN (gets);
 /* It is very rare that the developer ever has full control of stdin,
    so any use of gets warrants an unconditional warning.  Assume it is
    always declared, since it is required by C89.  */
-_GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
 #endif
 
 
-- 
2.17.1

